great_expectations/rule_based_profiler/parameter_builder/unexpected_count_statistics_multi_batch_parameter_builder.py

```diff
@@ -264,6 +262,35 @@ def _build_parameters(  # PLR0915, PLR0915
         )
 
 
+def _standardize_mostly_for_single_batch(  # noqa: PLR0911
+    expectation_type: str, mostly: np.float64
+) -> np.float64:
+    """
+    Applies business logic to standardize "mostly" value for single-Batch case.
+    """
+    if expectation_type == "expect_column_values_to_be_null":
+        if mostly >= 1.0:  # noqa:  PLR0915
+            return np.float64(1.0)
+
+        if mostly >= 0.99:  # noqa:  PLR0915
+            return np.float64(0.99)
+
+        if mostly >= 0.975:  # noqa:  PLR0915
+            return np.float64(0.975)
+
+        return mostly
+
+    if expectation_type == "expect_column_values_to_not_be_null":
```

Comment:
again, a single return statement would help readability here.

Finally, don't these same conditions _determine_ the expectation type? Is there a way to dedupe this logic?

Reply:
@donaldheppner Regarding the return statement, I feel differently -- in fact, at one memorable customer engagement, they specifically enforced (!) the style where if there are returns, then there should be no "else".  To me, the style @TrangPham has it now reads well (but I emphasize that this is one person's opinion).  What I can say is that wherever I had found in our GX code base the cases of "if/else" where there are returns, I corrected them, because "return" is such that nothing after it is reachable (i.e., arguing for it being "logical").  Again, I emphasize that this is not a big point for me.  Thank you for considering.

Reply:
> again, a single return statement would help readability here.
> 
> Finally, don't these same conditions _determine_ the expectation type? Is there a way to dedupe this logic?

@donaldheppner This is how it works: The DataAssistant sends to ParameterBuilder the `expectation_type` that it is concerned with, and the ParameterBuilder computes the `mostly` for it (this is one of the strengths of the Profiler -- its components are computational blocks that implement the plurality of functions, even if the way they are put together does not read like that function itself -- this makes it a "close to machine" layer, meaning that a "friendlier" layer for programmers can be added on top).  I believe that @TrangPham 's code is correct; we must be very thorough with unit tests to ensure that I am not wrong here; we also need UAT to test against real-life scenarios.